linux/hci/hci.go

```diff
@@ -505,6 +509,10 @@ func (h *HCI) handleLEConnectionComplete(b []byte) error {
 		}
 		h.params.RUnlock()
 	}
+	log.Println("incoming:", c.RemoteAddr())
```

Comment:
Please remove this log message. I think the library generally needs a logging pass.